repositories/memory/topics/get_topic_test.go

```diff
@@ -0,0 +1,88 @@
+package topics
+
+import (
+	"context"
+	"testing"
+
+	"github.com/matheusmosca/walrus/domain/entities"
+	"github.com/matheusmosca/walrus/domain/vos"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetTopic(t *testing.T) {
+	type args struct {
+		topicName vos.TopicName
+	}
+	type testCase struct {
+		description string
+		args        args
+		beforeRun   func(storage map[vos.TopicName]entities.Topic)
+	}
```

Comment:
You should add a wantErr to check the function output: [example](https://github.com/matheusmosca/walrus/blob/main/repositories/memory/topics/create_topic_test.go#L21)

Reply:
ACK. Will make the change